### PR TITLE
Remove case property aggregation from case exploder page

### PR DESCRIPTION
**Pull request by proteusvacuum**
This is causing the exploder page to 500 due to timeouts, and this functionality wasn't very useful anyway. 

https://sentry.io/organizations/dimagi/issues/901349775

### CODE REVIEW

**Comment by proteusvacuum** [comment]
bump

**Review by orangejenny — APPROVED** [review]
Code looks fine, though it looks like people will no longer be able to easily find the cases they created to delete them, is that true?

**Comment by proteusvacuum** [comment]
@orangejenny you are right... and the page didn't work anyway. Now this is tested, and added back the ability to delete cases. 
This was probably broken since https://github.com/dimagi/commcare-hq/pull/21763/

File: corehq/apps/hqcase/templates/hqcase/explode_cases.html [hunk 1]
```diff
@@ -64,35 +59,16 @@ <h2><small>Use at your own risk 🙃</small></h2>
...
-                                        </form>
-                                    </td>
-                                </tr>
-                            {% endfor %}
-                        </tbody>
-
-                    </table>
+                    <form class="form" action="" method="POST">
+                        {% csrf_token %}
+                        <input id="explosion_id" name="explosion_id" type="text" value="{{ explosion_id }}" placeholder="cc_explosion_id" />
```
Comment (orangejenny) [inline corehq/apps/hqcase/templates/hqcase/explode_cases.html]: Where is `explosion_id` coming from? I don't see it in `get_context_data`.

Comment (proteusvacuum) [reply corehq/apps/hqcase/templates/hqcase/explode_cases.html]: Sorry about that.

**Review by orangejenny — APPROVED** [review]

**Pull request merged**